src/common/mockFactory.ts

```diff
@@ -164,14 +164,14 @@ export default class MockFactory {
                 blobs: MockFactory.createAzureBlobs(i),
             });
         }
-        return { containerItems: result };
+        return {containerItems: result};
     }
 
     /**
      * Creates fake IAzureCloudStorageOptions
      */
     public static createAzureOptions(): IAzureCloudStorageOptions {
-        return { sas: "sas" };
+        return {sas: "sas"};
```

Comment:
Same here.

Reply:
Got it and we will discuss the criteria of code style. Thanks